src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -6838,6 +6864,137 @@ TEST_P(convolution_depthwise_gpu_fsv16, depthwise_conv_b_fs_yx_fsv16)
                 }
 }
 
+INSTANTIATE_TEST_SUITE_P(convolution_depthwise_gpu_b_fs_yx_fsv16_x1,
+                        convolution_depthwise_gpu_fsv16_xy,
```

Comment:
nit  but please leave a note that this is to test "1D" dw conv or add such a suffix to the test case name.

Reply:
Added _1d_depthwise_convolution, thanks